cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1792,15 +1792,33 @@ endif()
 
 #------------------------------------------------------------------------------------
 if(webgui)
-  ExternalProject_Add(
-     OPENUI5
-     URL ${CMAKE_SOURCE_DIR}/gui/webdisplay/res/openui5.tar.gz
-     URL_HASH SHA256=0cde25387047163fe2ed5f32eb7224628be10c3b6afa07d0b2a42002543909c2
-     CONFIGURE_COMMAND ""
-     BUILD_COMMAND ""
-     INSTALL_COMMAND ""
-     SOURCE_DIR ${CMAKE_BINARY_DIR}/ui5/distribution)
-  install(DIRECTORY ${CMAKE_BINARY_DIR}/ui5/distribution/ DESTINATION ${CMAKE_INSTALL_OPENUI5DIR}/distribution/ COMPONENT libraries FILES_MATCHING PATTERN "*")
+  if(NOT "$ENV{OPENUI5DIR}" STREQUAL "" AND EXISTS "$ENV{OPENUI5DIR}/resources/sap-ui-core-nojQuery.js")
+     message(STATUS "openui5 - use from $ENV{OPENUI5DIR}")
+     file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/ui5)
+     execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink
```

Comment:
@linev just a small comment, can you please check here what will happen in case of make install?

Reply:
It is not dedicated for that - just keep possibility to tweak ROOT build procedure to use openui from other directory. Only for expert usage

Reply:
Let's add this explicitly in the status message and as well in Release notes :+1:

Reply:
I add few comments into SearchInstalledSoftware.cmake file